bikeshed/ReferenceManager.py

```diff
@@ -604,6 +618,8 @@ def __init__(self, text, ref):
     def __getattr__(self, name):
         if name == "for_":
             name = "for"
+        if name not in self.ref:
```

Comment:
Why add this?  It breaks the standard "raise an error if the name doesn't exist" behavior.